hack/testing/logging.sh

```diff
@@ -264,7 +293,16 @@ if [ "$ENABLE_OPS_CLUSTER" = "true" ] ; then
 else
     USE_CLUSTER=
 fi
-os::cmd::expect_success "./e2e-test.sh $USE_CLUSTER"
+# e2e-test runs checks which do not modify any data - safe to use
+# in production environments
+./e2e-test.sh $USE_CLUSTER
```

Comment:
os::cmd::expect_success is really only for commands that either run for a very short period of time, or don't have a lot of output.  It saves the stderr and stdout until the command returns, which is not useful when you want to tail the logs in real time (i.e. from jenkins). Our test scripts `e2e-test.sh` and the other `test-*` scripts run for a long period of time and produce a lot of output so it isn't really suitable to use  `expect_success`.  `expect_success` will time the command and tell you how much time it took, and will also annotate the output e.g. red for failure and green for success.